File: src/main/webapp/js/Integration/Quantity.js
function Quantity(value, units) {
	this.value = value;
	this.units = units;
};

Quantity.prototype.prettyPrint = function() {
	return "Value: " + this.value
	+ " Units: " + this.units;
};

Quantity.prototype.jsonPrint = function() {
	return { value: this.value,
	units: this.units
	};
};

module.exports = Quantity;

File: src/main/java/Entities/Quantity.java
package Entities;

public class Quantity extends Entity {

	private static final String UNITS = "units";
	private static final String VALUE = "value";
	private static final String QUANTITY = "Quantity";

	public Quantity(double value, String units) {
		super(QUANTITY, null, null);
		entity.put(VALUE, value);
		entity.put(UNITS, units);
	}
}